### PR TITLE
Japanese Support

### PR DESCRIPTION
**Pull request by axross**
Hi!

I want to use Japanese on this hubot script.

Please consider to merge this PR :)

### CODE REVIEW

**Comment by therealklanni** [comment]
@axross will you please `git rebase` the latest to your fork and push?

Also can you post here what the unicode characters are?

File: src/plusplus.coffee [hunk 1]
```diff
@@ -96,7 +96,7 @@ module.exports = (robot) ->
   robot.respond ///
     (?:erase )
     # thing to be erased
-    ([\s\w'@.-:]+?)
+    ([\s\w'@.-:\u3040-\u30FF\uFF01-\uFF60\u4E00-\u9FA0]*)
```
Comment (jabbink) [inline src/plusplus.coffee]: `]*`? I think that should be `]+?`?

**Comment by axross** [comment]
@therealklanni 

These Regex means Japanese standard characters Hiragana, Katakana and standard Kanji.

**Comment by therealklanni** [comment]
@axross thanks. I knew they were Japanese characters, just trying to understand which ones.

**Comment by ajacksified** [comment]
@axross this is great - could you rebase against master?

**Comment by axross** [comment]
merged master.

**Comment by therealklanni** [comment]
Published v1.3.0

**Comment by axross** [comment]
thanks! 👏

**Pull request merged**